pdns/dnsdist-lua.cc

```diff
@@ -1572,6 +1572,57 @@ vector<std::function<void(void)>> setupLua(bool client, const std::string& confi
       }
     });
 
+    g_lua.registerFunction<void(DNSQuestion::*)(std::string, std::string)>("setTag", [](DNSQuestion& dq, const std::string& strLabel, const std::string& strValue) {
+
+      if(dq.qTag == NULL)
```

Comment:
It would be a bit nicer to use `nullptr` instead of `NULL`, as the former can't be confused with an integer.